src/debug.c

```diff
@@ -303,6 +303,9 @@ void _al_trace_suffix(const char *msg, ...)
       return;
    }
 
+#ifdef __EMSCRIPTEN__
```

Comment:
Does emscripten support stderr? Could we use the ALLEGRO_IPHONE code for this?

Reply:
Yes, stderr goes to the javascript console only, while stdout goes to the that fancy text widget in the HTML pages. I guess the Allegro logs make more sense in the javascript console, allowing the HTML widget to be used for something more user centric. Changing it.